src/main/java/cubicchunks/asm/mixin/core/client/MixinRenderGlobal.java

```diff
@@ -170,9 +170,9 @@ public int getRenderChunkYPos(BlockPos pos) {
     @Overwrite
     private RenderChunk getRenderChunkOffset(BlockPos playerPos, RenderChunk renderChunkBase, EnumFacing facing) {
         BlockPos blockpos = renderChunkBase.getBlockPosOffset16(facing);
-        return MathHelper.abs(playerPos.getX() - blockpos.getX()) > this.renderDistanceChunks * 16 ? null :
-                MathHelper.abs(playerPos.getY() - blockpos.getY()) > this.renderDistanceChunks * 16 ? null :
-                        MathHelper.abs(playerPos.getZ() - blockpos.getZ()) > this.renderDistanceChunks * 16 ? null :
+        return MathHelper.abs(playerPos.getX() - blockpos.getX()) > this.renderDistanceChunks * Cube.SIZE ? null :
+                MathHelper.abs(playerPos.getY() - blockpos.getY()) > this.renderDistanceChunks * Cube.SIZE ? null :
+                        MathHelper.abs(playerPos.getZ() - blockpos.getZ()) > this.renderDistanceChunks * Cube.SIZE ? null :
```

Comment:
While it removes a magic constant here, I think this part doesn't really allow for this value to be changed. Rendering will always stay 16x16x16 because anything bigger would add noticeable delays. So keep that in mind. Using 32x32x32 cubes is never going to be just changing a constant, you will need at least `VANILLA_SIZE=16` and `SIZE=32`, and using `VANILLA_SIZE` here